api-tests-2.4/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Maven
```

Comment:
I think we need a .gitignore in this project and to not commit these build artifacts from target folder.  Applies to the files listed below here as well...